src/compiler/tsc.ts

```diff
@@ -419,7 +417,7 @@ module ts {
     }
 
     function printVersion() {
-        sys.write(getDiagnosticText(Diagnostics.Version_0, version) + sys.newLine);
+        sys.write(getDiagnosticText(Diagnostics.Version_0, ts.version) + sys.newLine);

```

Comment:
Why do you need this namespace qualifier? You should have access to all the exports in the ts module.